maths/average_mode.py

```diff
@@ -1,34 +1,28 @@
-def mode(input_list: list) -> list:  # Defining function "mode."
+from collections import Counter

```

Comment:
I think the point was to show how to do it, not to hide how to do it inside a Counter.

Reply:
Alright, I'll keep the old method